server/src/types/group.types.ts

```diff
@@ -0,0 +1,17 @@
+import { Types } from 'mongoose';
+import { MongoInjectedFields } from './mongo.types';
+
+type GroupSchema = {
+  groupName: string;
+  userIds: Types.Array<Types.ObjectId>;
+  setlistIds: Types.Array<Types.ObjectId>;
+  createdBy: string;
+  lastUpdatedBy: string;
+  createdAt: Date;
```

Comment:
was wondering if this createdAt and lastUpdatedAt will interfere with the fields created by timestamps = true?

Reply:
no, because this should be user id. which actually reminds me it should be ObjectId LOL